RoboREPL.roboFontExt/lib/roboREPL.py

```diff
@@ -325,6 +330,57 @@ def saveTheme(self, name):
         userThemes[name] = theme
         setDefaultValue("userThemes", userThemes)
 
+    def exportSettings(self):
+        if inRoboFont:
+            exportPath = mojo.UI.PutFile(message="Export RoboREPL Settings", fileName="Settings.roboREPLSettings")
+        else:
+            exportPath = vanilla.dialogs.putFile(messageText="Export RoboREPL Settings", fileName="Settings.roboREPLSettings")
```

Comment:
I guess you can use vanilla.dialogs even if its runs inside RF.

Reply:
Ok, for some reason before I got an error when using vanilla.dialogs, but on trying it again it works!